library/src/main/java/no/finntech/capturandro/Capturandro.java

```diff
@@ -70,7 +66,6 @@ public Capturandro build() {
     Capturandro(Builder builder) {
         super();
         this.context = builder.context;
-        applicationContext = context.getApplicationContext();
         this.capturandroCallback = builder.capturandroCallback;
```

Comment:
We need a callback since we risk the activity getting dumped out of memory while we're away. I've updated the PR to include savedInstanceState handling.